README.md

```diff
@@ -2,33 +2,40 @@
 
 Welcome!
 
-![OpenXR-Unity-MixedReality-Samples-MainMenu-Cropped](Readme/OpenXR-Unity-MixedReality-Samples-MainMenu-Cropped.jpg)
+![OpenXR-Unity-MixedReality-Samples-MainMenu](Readme/OpenXR-Unity-MixedReality-Samples-MainMenu.jpg)
 
-These sample projects showcase features provided in the [Mixed Reality OpenXR Plugin for Unity](https://aka.ms/openxr-unity).
+These sample projects showcase how to build Unity applications for HoloLens 2 or Mixed Reality headsets using OpenXR plugin. For more details on installing related tools and setup Unity project, please reference [https://docs.microsoft.com/](https://aka.ms/openxr-unity).
```

Comment:
nit: I'd previously used the full name "Mixed Reality OpenXR Plugin", since searching for "OpenXR plugin" gives Unity's plugin as the first result and this more specific plugin as the second one. If you think this is fine, then it's probably not a big issue.
```suggestion
These sample projects showcase how to build Unity applications for HoloLens 2 or Mixed Reality headsets using the Mixed Reality OpenXR plugin. For more details on installing related tools and setting up a Unity project, please reference [the plugin documentation on https://docs.microsoft.com/](https://aka.ms/openxr-unity).
```

Reply:
fixed